### ISSUE TITLE
Presets Panel: Incorrect Role, "Menu Item," in one of panel's containers

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#5447]
## Issue Description
The container that holds the text, Clear all, and Import buttons in the Presets dialog is described programmatically as a `menuitem`, but it does not have the UI expected of this role. This can confuse screen reader users who rely on the role description to understand what an interactive element will do or how to interact with it.

Derived from #5187 .

## WCAG Criteria
[1.3.1 Info and Relationships - A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Notes
Remove the `role` attribute, as this section does not need a role, and remove the `tabindex`, as it is not an interactive element that needs keyboard focus and does not need to receive focus for itself.

## Screenshot or Screen Recording

<img width="1255" alt="Screen shot of the dev tools inspector describing the text as a menu item in the accessibility tree." src="https://github.com/user-attachments/assets/17490856-f131-4084-a940-ed4f42285fcf" />

## Relevant Code
```html
<div role="menuitem" class="pointer-none group m-1.5 flex h-8 min-w-[170px] gap-2 rounded px-5 py-2.5 !pr-3 text-sm !opacity-100 focus:ring-0 radix-disabled:pointer-events-none radix-disabled:opacity-50 md:min-w-[240px]" tabindex="-1">
  ...
</div>
```
